CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## [v2.5.3](https://github.com/infinum/enumerations/tree/v2.5.3) (2022-01-27)
```

Comment:
The date should be updated before release.

Reply:
Thanks! 😅